network-api/networkapi/buyersguide/factory.py

```diff
@@ -197,9 +206,120 @@ class Meta:
     easy_to_learn_and_use_helptext = Faker('sentence')
 
 
+class BuyersGuidePageFactory(PageFactory):
+
+    class Meta:
+        model = BuyersGuidePage
+
+
+class SoftwareProductPageFactory(PageFactory):
+
+    class Meta:
+        model = SoftwareProductPage
+
+    title = Faker('sentence')
+    handles_recordings_how = Faker('sentence')
+    recording_alert = LazyFunction(get_extended_yes_no_value)
+    recording_alert_helptext = Faker('sentence')
+    medical_privacy_compliant = Faker('boolean')
+    medical_privacy_compliant_helptext = Faker('sentence')
+    host_controls = Faker('sentence')
+    easy_to_learn_and_use = Faker('boolean')
+    easy_to_learn_and_use_helptext = Faker('sentence')
+    handles_recordings_how = Faker('sentence')
+    recording_alert = LazyFunction(get_extended_yes_no_value)
+    recording_alert_helptext = Faker('sentence')
+    medical_privacy_compliant = Faker('boolean')
+    medical_privacy_compliant_helptext = Faker('sentence')
+    host_controls = Faker('sentence')
+    easy_to_learn_and_use = Faker('boolean')
+    easy_to_learn_and_use_helptext = Faker('sentence')
+    first_published_at = Faker('past_datetime', start_date='-2d', tzinfo=timezone.utc)
+    last_published_at = Faker('past_datetime', start_date='-1d', tzinfo=timezone.utc)
+    slug = None
```

Comment:
If this is required, let's leave a code comment explaining why we need to explicitly set this to `None`.

Reply:
Ahh yes. Remnants of a factory thing I was testing. Its not needed. Will fix in the next commit.